[2022-04-11 20:06:42,743] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: schedule_data.rm_task scheduled__2016-01-01T00:00:00+00:00 [queued]>
[2022-04-11 20:06:42,779] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: schedule_data.rm_task scheduled__2016-01-01T00:00:00+00:00 [queued]>
[2022-04-11 20:06:42,780] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 20:06:42,780] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-11 20:06:42,780] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 20:06:42,809] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): rm_task> on 2016-01-01 00:00:00+00:00
[2022-04-11 20:06:42,818] {standard_task_runner.py:52} INFO - Started process 358985 to run task
[2022-04-11 20:06:42,851] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'schedule_data', 'rm_task', 'scheduled__2016-01-01T00:00:00+00:00', '--job-id', '1983', '--raw', '--subdir', 'DAGS_FOLDER/nfl_to_gcs.py', '--cfg-path', '/tmp/tmpqlxxw3_a', '--error-file', '/tmp/tmpwdnxv9z0']
[2022-04-11 20:06:42,855] {standard_task_runner.py:77} INFO - Job 1983: Subtask rm_task
[2022-04-11 20:06:43,142] {logging_mixin.py:109} INFO - Running <TaskInstance: schedule_data.rm_task scheduled__2016-01-01T00:00:00+00:00 [running]> on host 4ed3eda454fd
[2022-04-11 20:06:43,347] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=schedule_data
AIRFLOW_CTX_TASK_ID=rm_task
AIRFLOW_CTX_EXECUTION_DATE=2016-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2016-01-01T00:00:00+00:00
[2022-04-11 20:06:43,348] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-11 20:06:43,354] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm "/opt/***/schedule_data_2016.parquet"']
[2022-04-11 20:06:43,449] {subprocess.py:85} INFO - Output:
[2022-04-11 20:06:43,465] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-11 20:06:43,603] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=schedule_data, task_id=rm_task, execution_date=20160101T000000, start_date=20220411T200642, end_date=20220411T200643
[2022-04-11 20:06:43,703] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-11 20:06:43,798] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
